zip-0226.rst

```diff
@@ -57,15 +57,17 @@ issuance and bridging purposes.
 
 Overview
 ========
-In order to be able to represent different Asset identifiers in Orchard, we need to define a new data field in a note that uniquely represents the identifier of the Asset in question, which we call :math:`\mathsf{AssetId^{Orchard}}`.
+In order to be able to represent different Assets, we need to define a data field that uniquely represents the Asset in question, which we call the Asset Identifier :math:`\mathsf{AssetId}`. 
+This Asset Identifier is stored in Orchard-based ZSA notes as an Asset Base :math:`\mathsf{AssetBase}^{\mathsf{Orchard}}`.
```

Comment:
This Asset Identifier maps to an Asset Base :math:`\mathsf{AssetBase}^{\mathsf{Orchard}}` that is stored in Orchard-based ZSA notes.

Reply:
Done